### PR TITLE
Various table improvements

### PR DESCRIPTION
**Pull request by kschiffer**
#### Summary
This PR improves the UX around data displayed in tables in the Console in various ways.

<details><summary>See screenshots</summary>

image
image
image
image
image

</details>

#### Changes
- Improve column widths across entity tables
- Add `created_at` column to several entity tables
- Sort by `created_at` (latest first) by default (except for collaborators)
- Add sorting functionality for API keys, collaborators, pubsubs and webhooks
- Streamline right-tags inside API key and collaborator tables
  - Use localized right description per right
  - Use tooltips for overflowed content
  - Use tooltips for omitted rights counter
- Add number of devices per application entry in the applications table
  - Extend store logic to retrieve device count for list action
- Add number of collaborators per organization in the organizations table
  - Extend store logic to retrieve collaborator count for list action
- Add missing RPC fields and backend functionality for sorting API keys
- Small related styling fixes

#### Testing

Manual and existing e2es.

##### Regressions

None.

#### Notes for Reviewers
Sorting for collaborators is now solved in the frontend using a `limit` of `1000` and disabling sorting if the total count is above that. It felt like an acceptable compromise to me after looking into the complexity of sorting in the backend that uses a composed join of orgs and users which makes sorting quite complicated (to me).

#### Checklist

- [x] Scope: The referenced issue is addressed, there are no unrelated changes.
- [x] Compatibility: The changes are backwards compatible with existing API, storage, configuration and CLI, according to the compatibility commitments in `README.md` for the chosen target branch.
- [x] Documentation: Relevant documentation is added or updated.
- [x] Changelog: Significant features, behavior changes, deprecations and fixes are added to `CHANGELOG.md`.
- [x] Commits: Commit messages follow guidelines in `CONTRIBUTING.md`, there are no fixup commits left.

### CODE REVIEW

**Review by htdvisser — CHANGES_REQUESTED** [review]
Blocking this PR on a proper sorting implementation in the backend. I'll try to pick that up somewhere this week.

File: CHANGELOG.md [hunk 1]
```diff
@@ -15,8 +15,11 @@ For details about compatibility between different releases, see the **Commitment
 
 ### Changed
 
-- Entities are now fully validated when updated in the stores.
```
Comment (adriansmares) [inline CHANGELOG.md:20]: Don't drop the old changelog.

File: pkg/webui/console/containers/pubsubs-table/index.js [hunk 2]
```diff
@@ -106,12 +109,14 @@ export default class PubsubsTable extends React.Component {
...
       <FetchTable
         entity="pubsubs"
+        defaultOrder="-ids.pub_sub_id"
         addMessage={sharedMessages.addPubsub}
         headers={headers}
         getItemsAction={this.getPubsubsList}
         baseDataSelector={this.baseDataSelector}
         tableTitle={<Message content={sharedMessages.pubsubs} />}
         getItemPathPrefix={getItemPathPrefix}
+        handlesSorting
```
Comment (adriansmares) [inline pkg/webui/console/containers/pubsubs-table/index.js:120]: This should be `paginated=false` too, right ?

**Review by adriansmares — APPROVED** [review]

**Review by htdvisser — APPROVED** [review]

**Review by ryaplots — APPROVED** [review]
code-owned changes lgtm

**Review by nicholaspcr — APPROVED** [review]
Approving since the code-owned files changes were reverted.

Important question @htdvisser
>Blocking this PR on a proper sorting implementation in the backend. I'll try to pick that up somewhere this week.

Was it done already? Or is it being added on the rewrite?
I wanted to read it but I couldn't find the PR.

**Comment by htdvisser** [comment]
This was indeed done. You don't see my commits because @kschiffer squashed them into his original commits.

**Review by htdvisser — CHANGES_REQUESTED** [review]
Actually, after taking another look, it seems my commits were dropped from this PR.

**Review by htdvisser — APPROVED** [review]

**Pull request merged**